packages/workflow/src/errors/application.error.ts

```diff
@@ -7,7 +7,7 @@ export type ReportingOptions = {
 } & Pick<Event, 'tags' | 'extra'>;
 
 export class ApplicationError extends Error {
-	readonly level: Level;
+	level: Level;
```

Comment:
Thank you, agree! I had to do this for now to avoid a larger refactor in many child classes. Still plenty of tech debt to clean up in the error hierarchy.